skimage/data/_binary_blobs.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from ..filters import gaussian_filter
+
+
+def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
+                 volume_fraction=0.5, seed=None):
+    """
+    Generate synthetic binary image with several blob-like rounded objects.
```

Comment:
Change to: "several rounded blob-like objects." (English is a weird language, no one has been able to give me a good rule for how to determine the order of words in a case like this--so, I guess we have to go by feeling.)